src/frontend/helpers/library.ts

```diff
@@ -196,6 +196,16 @@ const launch = async ({
   showDialogModal
 }: LaunchOptions): Promise<{ status: 'done' | 'error' }> => {
   if (hasUpdate) {
+    const { ignoreGameUpdates } = await window.api.requestGameSettings(appName)
+
+    if (ignoreGameUpdates) {
+      return window.api.launch({
+        appName,
+        runner,
+        launchArguments: '--skip-version-check'
```

Comment:
I understand gogdl does not support this flag, is it ok to send it anyway? should we ignore this setting for gog games if we don't have a way to skip updates?

Reply:
True, forgot about this. Will fix.